aws_lambda_builders/__main__.py

```diff
@@ -46,6 +49,22 @@ def _error_response(request_id, http_status_code, message):
     })
 
 
+def _parse_version(version_string):
+
+    if VERSION_REGEX.match(version_string):
+        return float(version_string)
+    else:
+        raise ValueError("Protocol Version does not match : {}".format(VERSION_REGEX.pattern))
```

Comment:
Can you LOG.debug here? We should be able to debug this

Reply:
sure